packages/api/src/external/fhir/shared/index.ts

```diff
@@ -9,6 +9,7 @@ import { Operator } from "@medplum/core";
 export enum ResourceType {
   Organization = "Organization",
   Patient = "Patient",
+  Practitioner = "Practitioner",
   DocumentReference = "DocumentReference",
 }
```

Comment:
I actually suggest we deprecate this enum. AFAIK we can just use literals and have TS validate types as expected, and this is an incomplete list anyways.

Reply:
Sounds good 👍

Reply:
likewise, resolved but not updated